tests/integration/telemetry/stackdriver/testdata/server_access_log.json.tmpl

```diff
@@ -1,7 +1,7 @@
 {
     "http_request": {
         "request_method": "POST",
-        "request_url": "http://srv.{{ .EchoNamespace }}.svc.cluster.local:7070/proto.EchoTestService/Echo",
+        "request_url": "http://srv.{{ .EchoNamespace }}.svc.cluster.local/proto.EchoTestService/Echo",
```

Comment:
this changed because the test framework used to add port in the Host header which isn't representative of real world typical usage. No tracing code changed